src/client/components/Sidebar/ListPane/FlagCard.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+// App Imports
+import { phraseType } from '../../../types'
+import { calculatePhraseColor } from '../../../helpers'
+
+// Styles
+
+const Wrapper = styled.div`
+  border: 1px solid black;
+  width: 100%;
+  height: 82px;
+  padding: 15px;
+  margin-top: 20px;
+  box-shadow: 2px 2px 0 0 ${props => calculatePhraseColor(props.phrase)};
+`
+
+const FlagText = styled.div`
+  font-family: "Roboto Slab";
+  font-size: 22px;
+  font-weight: 300;
+`
+
+// Component
+class FlagCard extends React.Component {
```

Comment:
My linter screams at me for not using functions for stateless components. (Evidently they're more performant.) Consider rewriting these, it's pretty easy. This would become:

```
const FlagCard = (props) => {
  const { flaggedPhrase } = this.props
  return (
    …
  )
}
FlagCard.propTypes = {
  flaggedPhrase: phraseType,
}
```
With everything else being the same.